packages/ethereum/src/index.ts

```diff
@@ -17,24 +17,32 @@ import {
   READ_BALANCE,
   READ_BALANCE_ARRAY,
   BROADCAST,
+  READ_TOKEN_ID,
+  READ_OWNER,
+  READ_ASSET,
 } from "./actions";
 import Web3 from 'web3';
 // import * as ERC20_ABI from './erc20_abi.json'
 
 require("dotenv").config();
 
 const ERC20_ABI = require('./erc20_abi.json');
+const ERC721_ABI = require('./erc721_abi.json');
```

Comment:
Prefer imports over require in TS...

Reply:
I'm importing a json, I think that I used `require` because I couldn't make it work with `import`...